packages/ranges/src/elements/MultiThumbRange.js

```diff
@@ -0,0 +1,362 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+/* stylelint-disable block-no-empty */
+
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import styled from 'styled-components';
+import debounce from 'lodash.debounce';
+import { KEY_CODES } from '@zendeskgarden/react-selection';
+import { withTheme, isRtl } from '@zendeskgarden/react-theming';
+import RangeStyles from '@zendeskgarden/css-forms/dist/range.css';
+
+/**
+ * These Styled components are not exported with the other Views due to their logic
+ * being more tightly coupled with this specific implemenation.
+ */
+const StyledSlider = styled.div.attrs({
+  'data-test-id': 'slider',
+  className: props =>
+    classNames(RangeStyles['c-range__slider'], {
+      [RangeStyles['is-disabled']]: props.isDisabled,
+      [RangeStyles['is-rtl']]: isRtl(props)
+    })
+})`
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+  }
+`;
+
+const StyledTrack = styled.div.attrs({
+  'data-test-id': 'track',
+  className: RangeStyles['c-range__slider__track']
+})``;
+
+const StyledTrackRail = styled.div.attrs({
+  'data-test-id': 'rail',
+  className: RangeStyles['c-range__slider__track__rail']
+})``;
+
+const StyledThumb = styled.div.attrs({
+  'data-test-id': 'thumb',
+  className: props =>
+    classNames(RangeStyles['c-range__slider__track__rail__thumb'], {
+      [RangeStyles['is-focused']]: props.isFocused
+    })
+})``;
+
+class MultiThumbRange extends PureComponent {
```

Comment:
Technically we still support react 0.14.0 and `PureComponent` came in v15.3.0.

Reply:
I've updated this to use the standard `Component`. I was already checking for extra renders to help with `onChange` management so no other changes were needed.